### PR TITLE
PROVES-106 Add a paginator for search results

### PR DESCRIPTION
**Pull request by kehh**
Currently quicksearch is not returning results so needs a bit of work

### CODE REVIEW

**Comment by kehh** [comment]
Have created PROVES-113 to deal with quicksearch issue

File: app/lib/Plugins/SearchEngine/Elastic8.php [hunk 1]
```diff
@@ -336,10 +397,16 @@ public function search(
...
 		} catch (ClientResponseException $e) {
 			$this->getLogger()->logError($e->getMessage());
-			$results = ['hits' => ['hits' => []]];
+			$results = ['hits' => ['hits' => [], ['total' => ['value' => 0]]]];
 		}
-
-		return new WLPlugSearchEngineElastic8Result($results['hits']['hits'], $subject_tablenum);
+		$hits = $results['hits']['hits'];
+		$hits = array_combine(array_column($hits, '_id'), $hits);
+		dump($hits);
```
Comment (bri-gaia) [inline app/lib/Plugins/SearchEngine/Elastic8.php]: probably don't need to commit the dump?

**Comment by bri-gaia** [comment]
One comment, otherwise LGTM

File: app/lib/Plugins/SearchEngine/Elastic8.php (hunk 1 above)
Comment (kehh) [reply app/lib/Plugins/SearchEngine/Elastic8.php]: great catch thanks!

Comment (kehh) [reply app/lib/Plugins/SearchEngine/Elastic8.php]: OK fixed

**Pull request merged**